packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -167,7 +167,8 @@ class AndroidSdk {
 
   AndroidSdkVersion get latestVersion => _latestVersion;
 
-  String get adbPath => getPlatformToolsPath(globals.platform.isWindows ? 'adb.exe' : 'adb');
```

Comment:
The nice thing about this approach is technically the user could still install it after the daemon starts up too.